docs/releases/v1.6.0.md

```diff
@@ -0,0 +1,76 @@
+# OPA Core Module Release 1.6.0
+
+Welcome to the latest release of `OPA` module of (`Kubernetes Fury
+Distribution`)[https://github.com/sighupio/fury-distribution] maintained by team
+SIGHUP.
+
+This latest release is an attempt on upgrading the components in the module to
+its latest stable release along with adding the tech preview of the latest
+kubernetes release `v1.23.0`.
+
+## Component Images 🚢
+
+| Component                   | Supported Version                                                                     | Previous Version |
+|-----------------------------|---------------------------------------------------------------------------------------|------------------|
+| `gatekeeper`                | [`v3.7.0`](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.7.0)       | `v3.6.0`         |
+| `gatekeeper-policy-manager` | [`v1.5.1`](https://github.com/sighupio/gatekeeper-policy-manager/releases/tag/v0.5.1) | `v1.5.0`         |
+
+> Please refer the individual release notes to get a detailed info on the
+> releases. Eventhough gatekeeper `v3.7.0` has no breaking changes, it is worth having a look.
+
+## Known Issues ⚠️
+
+- This release involves recreation of certain resources like daemonset, deployments, etc. We recommend deleting it with `--cascade=orphan`, to avoid changes being made to the pods.
+- We drop support of Kubernetes versions <= v1.19.x
+
+## Breaking Changes 💔
+
+- Now Gatekeeper watches also for `DELETE` events as well. If you have custom constraints *you have to* adapt them in order to handle this with something like the follow rego code:
+  ```go
+  operation := input.review.operation
+  any([ operation == "CREATE", operation == "UPDATE" ])
+  operation != "DELETE"
+  ```
+- Add a Constraint Template to protect namespaces for being deleted. If you want to avoid accidental deletion of namespace, add annotation to your namespace
+  ```yaml
+  annotations:
+  opa.sighup.io/indelible-ns: "yes"
+  ```
+
+  Otherwise to enable deletion use the annotation:
+
+  ```yaml
+  annotations:
+  opa.sighup.io/indelible-ns: "no"
+  ```
+
+  If you don't put any annotation, the default is to protect the namespace.
+
+  By default the constraint is disabled - while the constrainttemplate is deployed - , but you can enable it by removing the comment in the line in the [kustomization.yaml](../../katalog/gatekeeper/rules/constraints/kustomization.yaml)
```

Comment:
I'm having second thoughts on passing the `DELETE` actions by default. I'm currently more inclined to have the user enable it if it wants together with the constraint, being the first time we deliver this rule and that it won't be used by everyone, I think it can cause more headaches than solutions